tests/utils_test.py

```diff
@@ -0,0 +1,40 @@
+"""Unit tests for english language checker"""
+from libcipher import utils as utils
+import io
+import sys
+import unittest
+
+
+def get_marys_dictionary():
+        english_words = ["had", "a", "little", "lamb"]
+        text = str("\n".join(english_words))
+        if sys.version < '3':
+            dict_file = io.StringIO(unicode('\n').join(english_words))
+        else:
+            dict_file = io.StringIO("\n".join(english_words))
```

Comment:
If this library is about checking for English words, why would we use unicode strings in Python 2?